using-tensorflow.py

```diff
@@ -0,0 +1,12 @@
+from __future__ import print_function
+
+import tensorflow as tf
+
+# Create a Constant op
+hello = tf.constant('Hello, World!')
+
+# Start tf session
+sess = tf.Session()
```

Comment:
It seems this code has been written with respect to Tensorflow 1.x and that would not run correctly if being executed through Tensorflow 2.x . Kindly disable the 2.x execution. You can have a rough idea from the suggestion I'm dropping.
```suggestion
import tensorflow.compat.v1 as tf
tf.disable_v2_behavior()
# Create a Constant op
hello = tf.constant('Hello, World!')

# Start tf session
sess = tf.Session()
```

Reply:
Thanks @Goku-kun 
Tested on both Tensorflow v1 and v2, works well on both.